packages/pg-cursor/index.js

```diff
@@ -42,7 +43,8 @@ class Cursor extends EventEmitter {
   submit(connection) {
     this.state = 'submitted'
     this.connection = connection
-    this._portal = 'C_' + nextUniqueID++
+    // the portal name must start with a lowercase c, for some reason
+    this._portal = 'c_' + nextUniqueID++
```

Comment:
Just to verify, this is still the case after we debugged the true issue w/command complete?

Reply:
Yup, this is actually necessary the last time I checked. Making the c uppercase broke my code.